src/config/constants.ts

```diff
@@ -120,3 +120,9 @@ export const ENV = {
 };
 
 export const TREE_VIEW_MAX_WIDTH = 400;
+
+export const CATEGORY_COLORS = {
+  [CATEGORY_TYPES.DISCIPLINE]: '#bbdefb',
+  [CATEGORY_TYPES.LANGUAGE]: 'grey',
+  [CATEGORY_TYPES.LEVEL]: '#5050d2',
+};
```

Comment:
Does Typescript automatically infer that the type is `Record<CATEGORY_TYPE, string>` or is it `Record<string, string>`? E.g. is `CATEGORY_COLORS["foobar"]` allowed?

We may want to explicitly type it the former way, to ensure that the keyed access is always type-checked 😊